sympy/vector/operators.py

```diff
@@ -0,0 +1,154 @@
+from sympy.vector.coordsysrect import CoordSysCartesian
+from sympy.core import Basic
+from sympy.vector.vector import Vector
+from sympy.vector.deloperator import Del
+
+
+class Gradient(Basic):
+    """
+    Represents unevaluated Gradient.
+    
+    Examples
+    ========
+    
+    >>> from sympy.vector import CoordSysCartesian, Gradient
+    >>> R = CoordSysCartesian('R')
+    >>> s1 = R.x*R.y*R.z
+    >>> Gradient(s1, R)
+    (Derivative(R.x*R.y*R.z, R.x))*R.i git + (Derivative(R.x*R.y*R.z, R.y))*R.j + (Derivative(R.x*R.y*R.z, R.z))*R.k
+    """
+
+    def __new__(cls, scalar, coord_sys):
+        return Del(coord_sys).gradient(scalar)
```

Comment:
This should not calculate the result, write something like:
```python
obj = Expr.__new__(...)
return obj
```
Same for the other classes.

Move calculation to `def doit(self, ...)` method.